examples/track.py

```diff
@@ -56,6 +53,7 @@ def run(args):
     
     model = YOLO(args['yolo_model'] if 'v8' in str(args['yolo_model']) else 'yolov8n')
     overrides = model.overrides.copy()
+    overrides['classes'] = args['classes']  # Add override to filter classes for yolov8n models
```

Comment:
Not needed, already overwritten in `combined_args = {**predictor.args.__dict__, **args}`

Reply:
Fixed.